cabal-install-solver/src/Distribution/Solver/Modular/Message.hs

```diff
@@ -258,8 +322,13 @@ showExposedComponent (ExposedExe name)               = "executable '" ++ unUnqua
 constraintSource :: ConstraintSource -> String
 constraintSource src = "constraint from " ++ showConstraintSource src
 
+-- FIXME: use a ANSI formating library like ansi-terminal to achieve this?
+-- bold :: String -> String
+-- bold str = "\ESC[1m" ++ str ++ "\ESC[0m"
+--
+-- The point would be to highlight the conflict in solver output:
```

Comment:
Ideally any user-facing pretty-printing should be done in `cabal-install` and not in `cabal-install-solver`